kafka/src/main/java/com/expedia/adaptivealerting/kafka/visualizer/Utility.java

```diff
@@ -0,0 +1,30 @@
+package com.expedia.adaptivealerting.kafka.visualizer;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Utility {
+
+
+    public static String convertToJson(Object object) {
+        String json = "";
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            json = objectMapper.writeValueAsString(object);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+        return json;
```

Comment:
If the exception occurs, then this will return an empty string, which isn't valid JSON. Better to simply throw an exception if the conversion fails since the client will have more context to respond. Feel free to wrap with a `RuntimeException` if you don't want to throw the original.